ntp-proto/src/nts_record.rs

```diff
@@ -2321,7 +2321,10 @@ mod test {
         let mut serverconfig = rustls::ServerConfig::builder()
             .with_safe_defaults()
             .with_client_cert_verifier(Arc::new(
-                rustls::server::AllowAnyAnonymousOrAuthenticatedClient::new(root_store.clone()),
+                #[cfg(not(feature = "nts-pool"))]
```

Comment:
Note: this is in test-code; it feels better (in the default mode for ntpd-rs) to run tests using a rustls-provided `ClientCertVerifier` instead of the home-brew one.